testing/trino-server-dev/etc/catalog/pinot.properties

```diff
@@ -0,0 +1,14 @@
+#
```

Comment:
What's the motivation of this change? There's no product test environment for Pinot at this time.

Reply:
I'm using this to test the connectivity with a local Pinot deployment. What do you mean by a test environment?

Reply:
If you can point me to an example, I can add a pinot test env as well

Reply:
"product test" means https://github.com/trinodb/trino/tree/master/testing/trino-product-tests https://github.com/trinodb/trino/commit/10909a411698f611ebf957c5400963c95c8e64cd is the example when we added a ClickHouse product test.